resources/serverless/values.yaml

```diff
@@ -64,7 +64,8 @@ global:
     app.kubernetes.io/instance: "{{ .Release.Name }}"
     app.kubernetes.io/version: "{{ .Values.global.images.function_controller.version }}"
     helm.sh/chart: '{{ include "chart" . }}'
-  dockerServicePort: 5000
```

Comment:
why changing the name of the port config ?

Reply:
Because it's confusing. It's not docker port, but it's registry port.